### DISCUSSION TITLE
Error using ListReport Extension

### DISCUSSION THREAD

**Discussion by leaguiar85** [marianfoo/ui5-cc-excelUpload#508] [General]
Hi Marian, 
Thanks for this amazing component! I´m trying to use it in an extension button in List Report.
![image](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/assets/54242167/ef7aee0b-687f-437e-bdf0-60c05581b80b)

Testing locally on VSCode works fine. But when I deploy and try to test in Launchpad, it cannot find the component usage in the manifest. I did a debug and found that in the createComponent, apparently, the application gets the manifest from List Report standard (that doesn't have componentUsages) instead of my manifest (but I'm not 100% sure).
![image](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/assets/54242167/ce0320cc-c9eb-49e9-9ff0-f769e1cae84f)

I've checked the version, and everything is fine.
![image](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/assets/54242167/cf2fa42a-9621-4924-8d9b-232aaa619a26)

![image](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/assets/54242167/1961f240-d1cd-4bb7-ab8b-a1b61ee2f560)

In the attachment, my manifest and controller.

Could you please help me to find out what I´m doing wrong?

Thanks in advance
Leandro

[ListReportExt.controller.txt](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/files/14460904/ListReportExt.controller.txt)
[manifest.json](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/files/14460906/manifest.json)
.

**Comment by marianfoo** [comment]
Hi @leaguiar85 
thank you for your report.
Please create a bug report here:
https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/issues/new?assignees=marianfoo&labels=bug%2Cexternal&projects=&template=bug.yml&title=%5BBug%5D%3A+
